src/styles/blocks/star.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--4 .stars__star:nth-child(-n+4) {
```

Comment:
what about  the rest, 1,2,3?

Reply:
Task describe only about 4 stars

Reply:
Yeah, you shouldn't display stars block with 1 or 5 active stars, just with 4 as you said

But that's a good practice to handle all variants in such cases - i.e. write CSS for each case. You might just reuse your code from `stars` task

I will not reject the task only because of this - you solved all other comments, but take it into account